src/pages/home/report/ReportActionCompose.js

```diff
@@ -447,9 +443,28 @@ class ReportActionCompose extends React.Component {
         const hasExceededMaxCommentLength = this.comment.length > CONST.MAX_COMMENT_LENGTH;
 
         return (
-            <View style={[shouldShowReportRecipientLocalTime && styles.chatItemComposeWithFirstRow]}>
-                {shouldShowReportRecipientLocalTime
-                    && <ParticipantLocalTime participant={reportRecipient} />}
+            <View style={[styles.chatItemComposeWithFirstRow]}>
+                <View style={[styles.flexRow, styles.chatItemComposeSecondaryRow, styles.chatItemComposeSecondaryRowOffset, styles.flexWrap]}>
+                    {shouldShowReportRecipientLocalTime
+                        && (
+                            <View style={[styles.mr3]}>
+                                <ParticipantLocalTime participant={reportRecipient} />
+                            </View>
+                        )}
+
+                    <View style={[styles.mr3]}>
+                        <YouAppearToBeOffline />
+                    </View>
+
+                    <View style={[styles.mr3]}>
+                        <ExceededCommentLength commentLength={this.comment.length} />
+                    </View>
+
+                    <View style={[styles.mr3]}>
+                        <ReportTypingIndicator reportID={this.props.reportID} />
+                    </View>
```

Comment:
Wrapping each of these components in a `<View style={[styles.mr3]}>` is going to produce inconsistent margins when, for example, the first and last components are visible, but the 2nd and 3rd return `null`. I think it'd be better to apply a `gap: 12px;` style to the flex container and eliminate these inner wrappers.

Also, NAB, but I think it reads cleaner to have `&& (` on the same line as the boolean var: `{shouldShowReportRecipientLocalTime && (`

Reply:
Oh, that's a really good point with the spacing. I like the idea of using `gap`!